pkgs/applications/misc/keystore-explorer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchzip, stdenv, jre }:
+
+stdenv.mkDerivation rec {
+  version = "5.4.4";
+  name = "keystore-explorer-${version}";
+  src = fetchzip {
+    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-544.zip";
+    sha256 = "01kpa8g6p6vcqq9y70w5bm8jbw4kp55pbywj2zrhgjibrhgjqi0b";
+  };
+
+  buildInputs = [ jre ];
+
+  installPhase = ''
+    substituteInPlace ./kse.sh --replace "java -jar" "${jre}/bin/java -jar"
+    mkdir -p $out/bin
+    cp -R ./* $out/
+    rm $out/kse.exe
+    mv $out/kse.sh $out/keystore-explorer.sh
+    ln -s $out/keystore-explorer.sh $out/bin/keystore-explorer
```

Comment:
Just put 1 `install -D kse.sh $out/bin/keystore-explorer`.

Reply:
Ended up having to directly write the script similarly to jd-gui: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/jd-gui/default.nix#L90